Day-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderImage = () => (
+  <div className="header-img-container">
```

Comment:
Since you are returning only the img HTML element, you don't need the div parent element.